google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -1768,7 +1784,7 @@ def iter(self):
         gcp = self.client
         if self.resource.container_api_enabled():
             for data in gcp.iter_container_clusters(
-                    projectid=self.resource['projectId']):
+                    projectid=self.resource['projectNumber']):
```

Comment:
Hmmm, this switch to `projectNumber` and still holding it as `projectid` might create some confusion, since in GCP, project id and project number are different from each other.  This would be better as:
`project_number=self.resource['projectNumber']`